src/lib/build-sprite.ts

```diff
@@ -0,0 +1,62 @@
+var spritezero = require('@mapbox/spritezero');
+var fs = require('fs');
+var glob = require('glob');
+var path = require('path');
+
+function generateLayoutAsync(option: any): Promise<any> {
+    return new Promise((resolve,reject) => {
+        spritezero.generateLayout(option,(err:any, result:object) => {
+            if (err) reject(err);
+            else resolve(result);
+        });
+    });
+}
+
+function generateImageAsync(option: any): Promise<any> {
+  return new Promise((resolve,reject) => {
+      spritezero.generateImage(option,(err:any ,result:object) => {
+          if (err) reject(err);
+          else resolve(result);
+      });
+  });
+}
+
+export async function buildSprite(svgPath: string, publicPath: string, iconSlug: string) {
+
+  const pxRatios = [1, 2]
```

Comment:
`sprite@4x.json` and `sprite@4xpng` are also required to generate.

Reply:
@JinIgarashi Thank you. I will add the function to generate `sprite@4x.json` and `sprite@4xpng`.